src/aind_data_transfer/writers.py

```diff
@@ -95,8 +115,22 @@ def copy_and_clip_data(src_dir, dst_dir, stream_gen, n_frames=100):
         videos_path = dst_dir / "Videos"
         videos_path_l = dst_dir / "videos"
         if os.path.isdir(videos_path):
-            new_videos_path = dst_dir / ".." / "videos"
-            shutil.move(videos_path, new_videos_path)
+            new_videos_path = dst_dir / ".." / "videos.zip"
```

Comment:
I think we should compress the videos individually and leave the file names the way they were. It's fine to be able to see file names here.

Reply:
Done